src/fastoad/cmd/api.py

```diff
@@ -132,6 +133,8 @@ def list_variables(
                               shells or if out parameter is not sys.stdout
     :param tablefmt: The formatting of the requested table. Options are the same as those available
                      to the tabulate package. See tabulate.tabulate_formats for a complete list.
+    :param variable_descriptions_format: if True and the out parameter is a file path, the file will
+                    use the variable_descriptions.txt format.
```

Comment:
2 points:
- Why `"and the out parameter is a file path"`? It should work the same if `out` is `sys.stdout`, shouldn't it?
- With `tablefmt`, that makes two arguments dedicated to output format. I would rather allow `tablefmt` to be also something like `var_desc` and manage this possibility in the code.

Reply:
Corrected in f5158e1 and e97877e.